File: URI Programs/C/1019.c
#include <stdio.h>

int main(){
	int n, hours = 0, minutes = 0;
	scanf("%d", &n);
	while(n >= 3600){
		n = n - 3600;
		hours = hours + 1;
	}
	while(n >= 60){
		n = n - 60;
		minutes = minutes + 1;
	}
	printf("%d:%d:%d\n", hours, minutes, n);
	return 0;
}

File: URI Programs/Python/1019.py
x1 = int(input())

hours = (x1)/3600
hours = int(hours)
x2 = x1 - hours*3600
mi = (x2)/60 
mi = int(mi)
x3 = x2 - mi*60
sec = x3

print(str(hours)+":"+str(mi)+":"+str(sec))